### PR TITLE
Sync db api into master.

### PR DESCRIPTION
**Pull request by xzdandy**
- [x] Code Synchronization
- [x] Test cases update

### CODE REVIEW

File: db_api_example.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    cursor = connection.cursor()
+    for onequery in query:
+        cursor.execute(onequery)
+        response = cursor.fetch_one()
+        print('Query: %s' % onequery)
+        print(response)
+
+if __name__ == '__main__':
+    if sys.argv[1] != 'sync':
+        asyncio.run(run_async(['INVALID QUERY',
```
Comment (gaurav274) [inline db_api_example.py:33]: These queries are obsolete, right? The Upload statement has changed paths, right?

File: src/models/server/response.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 class ResponseStatus(str, Enum):
     FAIL = -1
     SUCCESS = 0
+    MORE = 1
```
Comment (gaurav274) [inline src/models/server/response.py:10]: Can you add a comment to say why we have MORE? Thanks!

File: src/server/interpreter.py [hunk 3]
```diff
@@ -1,82 +1,68 @@
-# coding=utf-8
```
Comment (gaurav274) [inline src/server/interpreter.py:1]: Any reason for removing this header?

**Review by gaurav274 — CHANGES_REQUESTED** [review]
Thanks for adding the test cases!

**Comment by xzdandy** [comment]
Closing this PR. Address @gaurav274 's comments in #155

**Pull request closed without merging**